api/helm/api/values.yaml

```diff
@@ -46,6 +46,56 @@ varnish:
   pullPolicy: Always
   replicaCount: 1
 
+mercure:
+  enabled: true
+  acmeHosts: ""
+  allowAnonymous: "0"
+  corsAllowedOrigins: ""
+  debug: "0"
+  demo: "0"
+  jwtKey: "!UnsecureChangeMe!"
+  logFormat: "FLUENTD"
+  publishAllowedOrigins: ""
+  publisherJwtKey: ""
+  subscriberJwtKey: ""
+  replicaCount: 1
+  image:
+    #repository: registry.eu-gb.bluemix.net/mercure-saas/mercure-saas
```

Comment:
This should be removed (same for the other line). The public Docker image of Mercure open source must be used (Docker Hub: `dunglas/mercure`)